src/features/comments/commentsSlice.ts

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-param-reassign */
+
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { Comment } from '../../types/Comment';
+import { getPostComments } from '../../api/comments';
+
+export const fetchComments = createAsyncThunk<Comment[], number>(
+  'comments/fetchComments',
+  postId => {
+    const response = getPostComments(postId);
+
+    return response;
+  },
+);
+
+const initialState: {
+  comments: Comment[];
+  loaded: boolean;
+  hasError: boolean;
+} = {
+  comments: [],
+  loaded: true,
```

Comment:
The initial state for `loaded` should probably be `false`, since the comments are not loaded until the `fetchComments` async thunk is fulfilled. Setting it to `true` initially might cause components to render as if data is loaded when it's not.